### PR TITLE
DPI-931 migrate documentation to new format

### PR DESCRIPTION
**Pull request by oysteinsigholt**
Documentation from [https://ruterno.github.io/ota-schemas/mqtt/index.html](https://ruterno.github.io/ota-schemas/mqtt/index.html) added

### CODE REVIEW

**Comment by oysteinsigholt** [comment]
The sidebar has grown significantly with the new content, should we reduce `subMaxLevel`?

File: docs/ota.md [hunk 1]
```diff
@@ -7,27 +7,873 @@ The messages are encoded as json payloads, and you can validate any payload by u
 Ruters repositories.  
 
 MQTT is used as the transport mechanism for all OTA messages. MQTT is a simple pub-sub based messaging protocol, that works on top of TCP/IP as well as on websockets. 
+Ruter has agreed with the operators to use only JSON in MQTT messages and not support Protobuf. The contents of the messages will therefore be defined by JSON schemas and these will be made available to the operators.
```
Comment (oysteinsigholt) [inline docs/ota.md]: Copied directly over, but somewhat redundant considering the paragraph above. Should we remove this/some of it?

File: docs/ota.md [hunk 2]
```diff
@@ -7,27 +7,873 @@ The messages are encoded as json payloads, and you can validate any payload by u
...
 
+?> This page is an update of an outdated OTA Messages wiki page that was based on version 1.0 of the requirements. Version 1.1 of the requirements introduced some changes and after the development process began, new needs were identified as documented here, partly for use in formulating a change order for the operators selected for routes in Romerike and the electric bus contract.
```
Comment (DeanRuter) [inline docs/ota.md]: Why is there a"?>" here?

File: docs/ota.md [hunk 3]
```diff
@@ -7,27 +7,873 @@ The messages are encoded as json payloads, and you can validate any payload by u
...
+
+## Summary
+The summary refers to topics on board the buses. See below for mapping of the local topic to the bridged topic.
+Changes noted in status are based on a comparison with v1.1 of the OTA Messages document. Direction is in/out of the bus.
+
+| Local topic                        | Change     | Direction   | Name                             | Comments                                                                                                                                                                                             |
+|------------------------------------|------------|-------------|----------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| signon/json                        |            | Out         | Start block                      |                                                                                                                                                                                                      |
```
Comment (DeanRuter) [inline docs/ota.md]: Is it possible/worth it to have the "Name" section as URIs that take you directly to the section in question? I remember you mentioned it was subdivided into messages, which I assume "Name" refers to?

File: docs/ota.md [hunk 4]
```diff
@@ -7,27 +7,873 @@ The messages are encoded as json payloads, and you can validate any payload by u
...
+```json
+{
+  "eventTimestamp": "2017-10-31T08:38:02.749Z",
+  "expiryTimestamp": "2017-10-31T08:38:47.749Z",
+  "ref": "RUT:StopPlace:03012453",
+  "zoneId": "2b-vest",
+  "message": {
+    "no": {
+      "title": "Ankommer"
+      "text": "Oslo sentralstasjon"
```
Comment (DeanRuter) [inline docs/ota.md]: Visual bug or something used besides semicolon?

File: docs/ota.md (hunk 2 above)
Comment (oysteinsigholt) [reply docs/ota.md]: This is a docsify feature that gives the blockquote a nice style

File: docs/ota.md (hunk 4 above)
Comment (oysteinsigholt) [reply docs/ota.md]: FIxing missing comma now

File: docs/ota.md (hunk 3 above)
Comment (oysteinsigholt) [reply docs/ota.md]: done

File: docs/ota.md [hunk 5]
```diff
@@ -8,26 +8,946 @@ Ruters repositories.
 
 MQTT is used as the transport mechanism for all OTA messages. MQTT is a simple pub-sub based messaging protocol, that works on top of TCP/IP as well as on websockets. 
 
+?> This page is an update of an outdated OTA Messages wiki page that was based on version 1.0 of the requirements. Version 1.1 of the requirements introduced some changes and after the development process began, new needs were identified as documented here, partly for use in formulating a change order for the operators selected for routes in Romerike and the electric bus contract.
```
Comment (eivindga) [inline docs/ota.md]: I think you can remove any references to "outdated wiki page" now.

File: docs/ota.md [hunk 6]
```diff
@@ -8,26 +8,946 @@ Ruters repositories.
...
+      "unit": "rpm",
+      "value": 854
+    }
+  ]
+}
+```
+
+#### Fields
+
+#### Telemetry
```
Comment (eivindga) [inline docs/ota.md]: is this supposed to be a sub-header to "fields"?

Comment (oysteinsigholt) [reply docs/ota.md]: good catch!

**Comment by eivindga** [comment]
@oysteinsigholt You think you could change index.html and set the: 

      subMaxLevel: 3,

so that the left menu is a bit easier to read?

**Comment by oysteinsigholt** [comment]
> @oysteinsigholt You think you could change index.html and set the:
> 
> ```
>   subMaxLevel: 3,
> ```
> 
> so that the left menu is a bit easier to read?

added sidebar collapse plugins so that the menu is easier to read without losing granularity.

**Pull request merged**